creditcoin-js/src/extrinsics/extrinsics.ts

```diff
@@ -105,6 +106,9 @@ export const extrinsics = (api: ApiPromise) => {
 
     const exemptLoan = (dealOrderId: DealOrderId, lender: KeyringPair) => exemptLoanAsync(api, dealOrderId, lender);
 
+    const requestCollectCoins = (evmAddress: string, collector: KeyringPair, txHash: string) =>
```

Comment:
I am using `ExternalAddress` instead of `string`, see https://github.com/gluwa/creditcoin/pull/576/commits/c732e32220253c2583461cc343ee1ed1ce5ba037